packages/editor/src/components/rich-text/index.js

```diff
@@ -437,8 +437,10 @@ export class RichText extends Component {
 		const { keyCode } = event;
 
 		if (
-			( keyCode === BACKSPACE && isHorizontalEdge( rootNode, true ) ) ||
-			( keyCode === DELETE && isHorizontalEdge( rootNode, false ) )
+			window.getSelection().isCollapsed && (
```

Comment:
Should we continue with setting it at the top as browser dependencies? `const { getSelection } = window;`